### PR TITLE
Add fallback merchant account

### PR DESCRIPTION
**Pull request by daliwei**
cc @pierre @sbrossie for review.

### CODE REVIEW

File: src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java [hunk 1]
```diff
@@ -135,6 +137,15 @@ public AdyenConfigProperties(final Properties properties, final String currentRe
 
         this.merchantAccounts = properties.getProperty(PROPERTY_PREFIX + "merchantAccount");
         refillMap(countryToMerchantAccountMap, merchantAccounts);
+        if(this.countryToMerchantAccountMap.containsKey(FALL_BACK_MERCHANT_ACCOUNT_KEY))
```
Comment (pierre) [inline src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java]: Could you fix the style (braces placement, spacing, etc.)?

File: src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java [hunk 2]
```diff
@@ -49,6 +49,7 @@
                                                                                                               "sepadirectdebit");
     // Period is a bit aggressive by default. SOFORT (directEbanking) payment can take up to 14 days.
     public static final String DEFAULT_OFFLINE_BANK_TRANSFER_PENDING_PAYMENT_EXPIRATION_PERIOD = "P7d";
+    public static final String FALL_BACK_MERCHANT_ACCOUNT_KEY = "FALLBACK";
```
Comment (pierre) [inline src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java]: Could you update the `README` and document this feature?

File: src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java [hunk 3]
```diff
@@ -238,10 +249,10 @@ public String getMerchantAccount(final String countryIsoCode) {
         if (countryToMerchantAccountMap.isEmpty()) {
             return merchantAccounts;
         } else if (countryIsoCode == null) {
-            // In case no country is specified, but the user configured the merchant accounts per country, take the first one
-            return countryToMerchantAccountMap.values().iterator().next();
+            // In case no country is specified, but the user configured the merchant accounts per country, take the fallback one or the first one.
+            return MoreObjects.firstNonNull(fallBackMerchantAccount, countryToMerchantAccountMap.values().iterator().next());
```
Comment (pierre) [inline src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java]: An NPE will now be thrown if that country isn't configured and I'm not sure the stacktrace will be that useful.

Could you handle that case specifically instead? Maybe throw an `IllegalStateException` with a message indicating the country?

File: src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java (hunk 2 above)
Comment (daliwei) [reply src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java]: Done in e80ba9b

File: src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java (hunk 1 above)
Comment (daliwei) [reply src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java]: Done in 
638f8a8

File: src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java (hunk 3 above)
Comment (daliwei) [reply src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java]: Good catch. Done in 36984ba

**Comment by pierre** [comment]
Thanks!

**Comment by pierre** [comment]
0.5.9 released.

**Comment by daliwei** [comment]
Thanks @pierre

**Pull request merged**